app/templates/courses/teacher-courses-view.jade

```diff
@@ -67,7 +67,7 @@ block content
                               | .
                             = " "
                             span
-                              = level.get('name').replace('Course: ', '')
+                              = i18n(level.attributes, 'name').replace('Course: ', '')
```

Comment:
Yeah, but if it doesn't work, it's still fine, just cosmetic, and we currently only have that one level that does this (Course: Winding Trail).